vagrant/pg_config.sh

```diff
@@ -1,7 +1,8 @@
 apt-get -qqy update
 apt-get -qqy install postgresql python-psycopg2
 apt-get -qqy install python-flask python-sqlalchemy
-apt-get -qqy install python-pip
+wget https://bootstrap.pypa.io/get-pip.py
+python ./get-pip.py
```

Comment:
Can we combine this into a single line and test if the wget completes? I think it's as simple as `wget https://bootstrap.pypa.io/get-pip.py && python ./get-pip.py` but would want you to test first.

If that works, squash into a single commit and I'll merge.